pset-iterators.js

```diff
@@ -65,6 +76,13 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 2:')
 
 // Add your code below this line
+  const presidentsNamedJames = presidents.filter(function(name) {
+  return name.president.match(/James/)
```

Comment:
@Ashbk32 nice work, line 80 should be indented since it is inside of the `.filter()` method